CMakeLists.txt

```
@@ -89,9 +89,9 @@ endif()
 
 #--- add version files ---------------------------------------------------------
 
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/EDM4HEPVersion.h
-               ${CMAKE_CURRENT_BINARY_DIR}/EDM4HEPVersion.h )
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/EDM4HEPVersion.h
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/EDM4hepVersion.h.in
+               ${CMAKE_CURRENT_BINARY_DIR}/EDM4hepVersion.h )
```

```suggestion
               ${CMAKE_CURRENT_BINARY_DIR}/edm4hep/EDM4hepVersion.h )
```
Maybe this should go into `edm4hep` as well? In order to be compatible with where we install it to.

---

Makes sense!